macos/xcode/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -14,16 +14,17 @@
 		8061BF7E22EF936800F2D245 /* DatePickerView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8061BF7C22EF936700F2D245 /* DatePickerView.swift */; };
 		8061BF7F22EF936800F2D245 /* DatePickerController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8061BF7D22EF936700F2D245 /* DatePickerController.swift */; };
 		8061BF8322EF957200F2D245 /* TestDatePickerController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8061BF8222EF957200F2D245 /* TestDatePickerController.swift */; };
-		80FEB30F230DE91D00F9FF3E /* OfficeUIFabric.strings in Resources */ = {isa = PBXBuildFile; fileRef = 80FEB2F8230DE53F00F9FF3E /* OfficeUIFabric.strings */; };
-		8F16282123CCD17900F81956 /* OfficeUIFabric.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8F16282023CCD17900F81956 /* OfficeUIFabric.xcassets */; };
+		80FEB30F230DE91D00F9FF3E /* FluentUI.strings in Resources */ = {isa = PBXBuildFile; fileRef = 80FEB2F8230DE53F00F9FF3E /* FluentUI.strings */; };
+		8F16282123CCD17900F81956 /* FluentUI.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8F16282023CCD17900F81956 /* FluentUI.xcassets */; };
```

Comment:
I posed another version of this question elsewhere, but do we want to push for things (mostly source files) in the FluentUI project to have _ios and _macos suffixes for easy CMD+Shift+O? Or is that a lot of change for very little benefit

Reply:
So this to me is a bit of a different question. These are internal implementation details to the framework and as such we can change them at-will later without client impact. Since we're currently still in an "xcodeproj per platform" configuration, I'd say leave these as-is for now and consider changing if we find issues later.